syntax = "proto3";
package comment;

service CommentService {
    rpc CreateComment(CreateCommentRequest) returns (CommentResponse);
    rpc GetComment(GetCommentRequest) returns (CommentResponse);
}

message CreateCommentRequest {
    string task_id = 1;
    string content = 2;
    string user_id = 3;
}

message CommentResponse {
    string comment_id = 1;
    string task_id = 2;
    string content = 3;
    string user_id = 4;
}

message GetCommentRequest {
    string comment_id = 1;
}
